src/styles/_stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url("./../images/star.svg");
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+}
+
+.stars--1 :first-child,
+
+.stars--2 :nth-child(-n+2),
+
+.stars--3 :nth-child(-n+3),
+
+.stars--4 :nth-child(-n+4),
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url("./../images/star-active.svg");
+}
```

Comment:
Try to fix it
<img width="785" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/43c53008-f240-494f-90dc-8f25590c68f4">